lib/chef_zero/endpoints/organization_policies_endpoint.rb

```diff
@@ -0,0 +1,99 @@
+
+module ChefZero
+  module Endpoints
+    # /organizations/NAME/policies
+    class OrganizationPoliciesEndpoint < RestBase
+      def set_opts
+        [ :create_dir, :data_store_exceptions ]
+      end
+
+      def hashify_list(list)
+        list.reduce({}) { |acc, obj| acc.merge( obj => {} ) }
+      end
+
+      def get(request)
+
+        # vanilla /policies.
+        if request.rest_path.last == "policies"
+          response_data = {}
+          policy_names = list_data_or_else(request, nil, [])
+          policy_names.each do |policy_name|
+            policy_path = request.rest_path + [policy_name]
+            policy_uri = build_uri(request.base_uri, policy_path)
+            revisions = list_data_or_else(request, policy_path + ["revisions"], {})
+
+            response_data[policy_name] = {
+              uri: policy_uri,
+              revisions: hashify_list(revisions)
+            }
+          end
+
+          return json_response(200, response_data)
+        end
+
+        # /policies/:policy_name
+        if request.rest_path[-2] == "policies"
+          if !exists_data_dir?(request)
```

Comment:
I was only dealing with the in-memory data store, which is just a giant JSON string. I believe there is some related ChefFS stuff to do, but it didn't crop up in this round.